test/spec/PalindromSpec.js

```diff
@@ -9,10 +9,10 @@ describe("Palindrom", function () {
   });
 
   describe("backward compatibility", function () {
-    it("should work if PuppetJs constructor is used", function (done) {
+    it("should work if Puppet constructor is used", function (done) {
       var initSpy = jasmine.createSpy();
 
-      this.palindrom = new PuppetJs({remoteUrl: window.location.href, callback: initSpy});
+      this.palindrom = new Puppet({remoteUrl: window.location.href, callback: initSpy});
```

Comment:
Yes, because there _was_ the same mistake in the backward compatibility code here:
https://github.com/Palindrom/Palindrom/pull/126/commits/d84db53b60bcc7b5b2f01799a0f80754d01764e0#diff-92b698a38d3db0381546f440a80e631bL799